### PR TITLE
Add support for mosquitto bridge versions 131 and 132

### PR DESCRIPTION
**Pull request by phil-mitchell**
These protocols are for bridge mode of version 3.  According to https://github.com/mqtt/mqtt.org/wiki/bridge_protocol these versions set the following properties:
- ras = true
- nl = true

### CODE REVIEW

**Review by mcollina — APPROVED** [review]
lgtm

**Comment by phil-mitchell** [comment]
I'm not sure it's clear that I don't have write access, so somebody will have to merge this if it's okay.

**Pull request merged**
